src/main/java/ca/on/oicr/gsi/dimsum/service/filtering/SampleSort.java

```diff
@@ -37,4 +39,17 @@ public Comparator<Sample> comparator() {
     return comparator;
   }
 
+  public static int getSampleQcStatus(Sample sample) {
```

Comment:
Change to protected. Getting an int to represent QC status doesn't really make sense outside of this class, so ideally would be private, but protected makes it visible for testing